### PR TITLE
07 bootstrap

### PR DESCRIPTION
**Pull request by mochammadilham06**
# Description Task

## Praktikum Bootstrap
[Link Soal](https://docs.google.com/document/d/1HlS8tK1b8HfobjdVQELTFv_GCKAkMOpjKXm9vYLSaw4/edit)

- [x] Membuat Navbar Responsive dengan `Bootstrap`
- [x] Membuat Main Content Responsive dengan `Bootstrap`
- [x] Membuat Footer Responsive dengan `Bootstrap`
- [x] Membuat Resume Materi `Bootstrap`

### CODE REVIEW

File: 06_CSS/praktikum/file1.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="container">
+        <div class="BoxRed">
+            <p>Saya adalah box red berukuran 100px</p>
+        </div>
+        <div class="BoxBlue">
+            <p>Saya adalah box blue berukuran 100px</p>
+        </div>
+        <div class="BoxGreen">
+            <p>Saya adalah box green berukuran 350px</p>
+        </div>
```
Comment (syukronarie) [inline 06_CSS/praktikum/file1.html:22]: classname convention kalo bisa diseragamkan.

**Review by syukronarie — APPROVED** [review]
Reviewed and approved

File: 07_Bootstrap/index.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        </div>
+        <div class="container text-center mt-5">
+            <div class="main-content row">
+                <div class="col-lg-4 col-md-4 col-sm-4">
+                    <div class="pict1">
+                        <img class="img-fluid mb-3" src="./assets/img/cloud.svg" alt="Clude Picture">
+                        <h5>INTRODUCING WHATSAPP</h5>
+                        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit</p>
+                        <a href="#">Read More</a>
+                    </div>
```
Comment (syukronarie) [inline 07_Bootstrap/index.html:80]: kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)

Comment (mochammadilham06) [reply 07_Bootstrap/index.html:80]: Siap makasih informasinya mas

**Pull request merged**